docs/src/main/asciidoc/README.adoc

```diff
@@ -11,7 +11,7 @@ For a deep dive into the project, refer to the Spring Cloud GCP Reference docume
 * link:https://googlecloudplatform.github.io/spring-cloud-gcp/1.2.x/reference/html/index.html[Spring Cloud GCP 1.2.x]
 * link:https://googlecloudplatform.github.io/spring-cloud-gcp/1.1.x/single/spring-cloud-gcp.html[Spring Cloud GCP 1.1.x]
 
-If you prefer to learn by doing, try taking a look at the https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/master/spring-cloud-gcp-samples[Spring Cloud GCP sample applications] or the https://codelabs.developers.google.com/spring[Spring on GCP codelabs].
+If you prefer to learn by doing, try taking a look at the https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/main/spring-cloud-gcp-samples[Spring Cloud GCP sample applications] or the https://codelabs.developers.google.com/spring[Spring on GCP codelabs].
```

Comment:
Oof - all of these links are going to break in the previous release documents. (see `gh-pages` branch).

cc @GoogleCloudPlatform/cloud-java-frameworks-team

Reply:
Will this be a problem once we fully move to `main` as the default?

Reply:
Assuming GH doesn't do any magical redirects, it would result in broken links in previously released documentation. For example, see the versioned subfolders here: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/gh-pages

That being said, a branch-wide find/replace would likely fix it up pretty easily, so I'm not too worried about this one.

Reply:
It looks like GH will do the redirect: https://github.blog/changelog/2020-07-17-links-to-deleted-branches-now-redirect-to-the-default-branch/

Reply:
Tested the redirects and it works!